README.md

```diff
@@ -36,7 +36,7 @@ Easy Mock wrapper for mocking EntityFrameworkCore 5 (EFCore5) DbContext and DbSe
 For the Moq version, you can use all known [Moq](https://github.com/Moq/moq4/wiki/Quickstart) features, since both `DbSetMock` and `DbContextMock` inherit from `Mock<DbSet>` and `Mock<DbContext>` respectively.
 
 ## Example usage
-
+```C#
     public class User
```

Comment:
Can you also unindent te code, otherwise it will show up as indented in the code-block.

Reply:
Sure, done.